securedrop_client/logic.py

```diff
@@ -520,8 +520,6 @@ def update_sources(self):
         Display the updated list of sources with those found in local storage.
         """
         sources = list(storage.get_local_sources(self.session))
-        if sources:
-            sources.sort(key=lambda x: x.last_updated)
```

Comment:
I wonder if no longer sorting by `last_updated` improves performance? If the server already provides the list in `last_updated` order and we always insert the sources in the local database in the same order then we can assume that `get_local_sources` provides the source list in `last_updated` order. Is that your thinking?

Reply:
I modified `get_local_sources` to provide the list in `last_updated` order in SQL (other places we use `get_local_sources` the ordering doesn't matter).

Reply:
i see the change now. thanks for the explanation.